src/Components/ExistingEvents.js

```
@@ -0,0 +1,70 @@
+        onCancel={handleCancel}
+      >
+        <List
+          grid={{ gutter: 16, column: 1 }}
+          dataSource={existingEvents}
+          renderItem={item => (
+            <List.Item>
+              <Card title={item.eventname}>
+                <p>시작: {item.startday}</p>
+                <p>종료: {item.endday}</p>
+                <Button type="primary" onClick={() => window.location.href = `http://localhost:8080/test/?key=${item.uuid}`}>
+                  일정 바로가기
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)